vcd/resource_vcd_network_isolated.go

```diff
@@ -82,7 +85,7 @@ func resourceVcdNetworkIsolated() *schema.Resource {
 			},
 
 			"dhcp_pool": &schema.Schema{
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
Is there any risk we get into ordering trouble - if user has 2+ dhcp pools (same thoughts about 'static_ip_pool') defined. Are we sure this order does not change during multiple reads. If so, `terraform apply` would tell them to recreate all pools randomly as the order matters for `TypeList`.

Also thinking that the addresses of pools will change for users. They have `resource.dhcp_pool.XXXX.start_address` (where XXXX is the hash), but it will become `resource.dhcp_pool.YY.start_address` (where YY is the index of array). Probably majority of them still convert this to list using `tolist()` as it would be a guesswork.

Reply:
Didn't list is Ordered?..
```
Used to represent an ordered collection of items, where the order the items are presented can impact the behavior of the resource being modeled. An example of ordered items would be network routing rules, where rules are examined in the order they are given until a match is found. The items are all of the same type defined by the Elem property.

Schema example:

"termination_policies": {
  Type:     schema.TypeList,
  Elem: &schema.Schema{
    Type: schema.TypeString,
  },
},
```

Reply:
Changed to Set. Please review again after next push